html-personal/html-forms_Finding-a11y-errors/index.html

```diff
@@ -16,16 +16,23 @@ <h1 class="title">Understanding A11y in Forms</h1>
         Please let us know how we can support you with a11y issues.
       </p>
 
-      <form for="title">
-        <fieldset>
+      <form aria-labelledby="title" for="title">
+        <fieldset aria-describedby="description">
           <legend>Enter your information</legend>
-          <label aria-describedby="name-input">Name:</label>
+
+          <label for="name-Input">Name:</label>
           <input id="name-Input" type="text" name="name" />
-          <label aria-describedby="age-input">Age:</label>
-          <input id="age-Input" type="text" name="age" />
-          <label aria-describedby="color-input">Color you cannot see:</label>
+
+          <label for="age-Input">Age:</label>
+          <input id="age-Input" type="number" min="1" max="99" name="age" />
+
+          <label for="color-Input" aria-describedby="color-input"
```

Comment:
Do we need the aria-label here? I thought we only need it in the <fieldset> and <legend>?

Reply:
to be honest it is not very clear to me. I assume that I can use aria whenever the element itself does not have a text compartment I can use to write text, which then makes it difficult to understand what this element is about. this is why I wrote aria here, so the screen reader identifies it as a `color input` element... but again, not 100% sure 👀